docs/secure.md

```diff
@@ -8,99 +8,68 @@
 
```

Comment:
question: do we also need to document how to rotate certificates? been a while since I looked at this, but I think you can do this by changing these values: https://github.com/mongodb/helm-charts/blob/main/charts/community-operator/values.yaml#L86-L87